src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbyplaintextcommentfilter/InputSuppressWithNearbyPlainTextCommentFilterCheckFormat.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+: '
+SuppressWithNearbyPlainTextCommentFilter
+commentFormat = (default)SUPPRESS CHECKSTYLE (\\w+)
+checkFormat = .*Line.*
+messageFormat = (default)(null)
+idFormat = (default)(null)
+influenceFormat = (default)0
+
+com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck
+eachLine = true
+fileExtensions = (default)
+
+com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
+fileExtensions = (default)all files
+ignorePattern = (default)^(package|import) .*
+max = (default)80
+
+
```

Comment:
I see that `continuous-integration/drone/pr` does not like this `.sh` input file:
```
+ ./.ci/checkchmod.sh

Expected mode for '.sh' files is 755.

Files that violates this rule:

./src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbyplaintextcommentfilter/InputSuppressWithNearbyPlainTextCommentFilterCheckFormat.sh
```

Shall I change it to something else?

Reply:
Hm, I see no need to make test input file executable. @stoyanK7 please investigate what we have done with other script file inputs (suppression or chmod)

Reply:
I meant actually rewriting it to some other file extension but that works as well. On it.

Reply:
Please add exclusion by most restrictive regex possible to checkchmod.sh, something like `^Input[A-Za-z0-9_*-]*\.sh`.

Reply:
Done. Drone seems to be passing `checkchmod`